tests/gha-run-k8s-common.sh

```diff
@@ -94,6 +94,25 @@ function install_kubectl() {
     sudo az aks install-cli
 }
 
+# Install the kustomize tool in /usr/local/bin if it doesn't exist on
+# the system yet.
+#
+function install_kustomize() {
+	local version
+	local arch
+
+	if command -v kustomize >/dev/null; then
+		return
+	fi
+
+	ensure_yq
+	version=$(get_from_kata_deps "externals.kustomize.version")
```

Comment:
It is not a big deal but I prefer put it the local here `local version` `local arch`